Microsoft.CAP.Portal/Microsoft.CAP.Portal/Microsoft.CAP.Portal.csproj

```diff
@@ -209,7 +215,6 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="App_Data\" />
-    <Folder Include="Models\" />
```

Comment:
Maybe because I add TuningViewModel and it is replaced by Models\TuningViewModel.cs in line 122